tests/flows/test_slurm_local.py

```diff
@@ -2,20 +2,68 @@
 import pytest
 
 @pytest.mark.eda
-@pytest.mark.skip(reason='Running slurm relies on Slurm account info that is injected by server, so it does not run locally.')
 def test_slurm_local_py(gcd_chip):
     '''Basic Python API test: build the GCD example using only Python code.
+       Note: Requires that the test runner be connected to a cluster, or configured
+       as a single-machine "cluster".
     '''
 
     # Inserting value into configuration
-    gcd_chip.set('jobscheduler', 'slurm')
+    gcd_chip.set('option', 'scheduler', 'slurm')
 
     # Run the chip's build process synchronously.
     gcd_chip.run()
 
     # Verify that GDS file was generated.
     assert os.path.isfile('build/gcd/job0/export/0/outputs/gcd.gds')
 
+@pytest.mark.eda
+def test_slurm_local_py_prefix(gcd_chip):
```

Comment:
Make these new tests unit tests on `Chip._deferstep`, not functional end-to-end tests.